src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/ViewModels/ImageResizerViewModel.cs

```diff
@@ -250,11 +250,11 @@ public string EncoderGuid
             }
         }
 
-        public void AddRow()
+        public void AddRow(string sizeName)
         {
             ObservableCollection<ImageSize> imageSizes = Sizes;
             int maxId = imageSizes.Count > 0 ? imageSizes.OrderBy(x => x.Id).Last().Id : -1;
-            ImageSize newSize = new ImageSize(maxId + 1);
+            ImageSize newSize = new ImageSize(maxId + 1, sizeName, ResizeFit.Fit, 854, 480, ResizeUnit.Pixel);
```

Comment:
Can we add a numerator (e.g. New size, New size (1), New size (2))?

You can probably re-use the FancyZones code. You'd need to loop through imageSizes instead.
https://github.com/microsoft/PowerToys/blob/c6a24b3907c34bf5edf68ee0be1696f38f430d5b/src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs#L164

Reply:
I suggest to create a common method that can be reused. Thoughts?

Where should I add the code for the method?